charts/lagoon-core/values.yaml

```diff
@@ -50,6 +50,8 @@
 
 # These values are the chart defaults, but can be overridden.
 
+k8up: false
```

Comment:
as these are just annotations and don't hurt, I would make them enabled by default.